fenicsprecice/fenicsprecice.py

```diff
@@ -273,9 +286,47 @@ def initialize(self, coupling_subdomain, mesh, function_space, write_function=No
             Recommended time step value from preCICE.
         """
 
-        coords = function_space.tabulate_dof_coordinates()
+        write_function_space, write_function = None, None
+        if type(write_object) is Function:
+            write_function_space = write_object.function_space()
+            write_function = write_object
+        elif type(write_object) is FunctionSpace:
+            write_function_space = write_object
+            write_function = None
```

Comment:
```suggestion
        if type(write_object) is Function:  # precice.initialize_data() will be called, since fenics.Function is provided by user
            write_function_space = write_object.function_space()
            write_function = write_object
        elif type(write_object) is FunctionSpace:  # only a FunctionSpace is provided by user. preCICE will use default zero values for initialization.
            write_function_space = write_object
            write_function = None
```
I think here a comment would be helpful in order to document what thes two branches actually mean: